### ISSUE TITLE
Disable VsVim shortcut doesn't work if there are errors in the code

### ISSUE THREAD

**Issue by vsviridov** [VsVim/VsVim#852]
If "Error List" contains any errors, then Ctrl+Shift+F12 will cycle between errors in the document. Once all errors are removed so that the VS doesn't complain about them, the shortcut kicks in and I can toggle VsVim state. Might be nice to just add a "disable" button next to "Options" button in the status bar.

**Comment by treestryder** [comment]
Ctrl+Shift+F12 is still handled by Visual Studio in v2015. It would be nice to have an option to disable VsVim, maybe with a mention of the key combo listed next to it.

When I first installed VsVim, I didn't know of the key combination and went looking for a setting. Eventually landing on the FAQ. Months later, I couldn't remember the key combination, so I again went looking for a setting. Again, finding none, I Googled to find it the FAQ. I try the key combination and my cursor starts jumping around. There was no mention of potential key clash. After more Googling, I found myself reading this entire thread. Long story short... The combination of my short term memory, no on/off setting in options, a clashing key combo, a FAQ entry that is missing a key bit of information; have made for a convoluted method to temporarily disable VsVim.
